Mathlib/MeasureTheory/Function/LpSpace.lean

```diff
@@ -270,6 +269,10 @@ protected theorem coe_nnnorm (f : Lp E p μ) : (‖f‖₊ : ℝ) = ‖f‖ :=
   rfl
 #align measure_theory.Lp.coe_nnnorm MeasureTheory.Lp.coe_nnnorm
 
+@[simp, norm_cast]
+theorem nnnorm_coe_ennreal (f : Lp E p μ) : (‖f‖₊ : ℝ≥0∞) = snorm f p μ :=
+  ENNReal.coe_toNNReal f.snorm_ne_top
```

Comment:
Ugh, sorry; I guess it's `<| Lp.norm_ne_top f`

Reply:
```suggestion
  ENNReal.coe_toNNReal <| Lp.snorm_ne_top f
```